app/models/application.rb

```diff
@@ -151,18 +152,13 @@ def full_name
   end
 
   def applicant_over_61?
-    applicant_age >= 61
+    applicant.age >= 61
```

Comment:
I'm thinking whether we do use the `age` somewhere outside this `applicant_over_61?` method. If not, than we don't actually need the delegation. I thought originally that we use the age directly.